_posts/2023-07-10-trino-fest-2023-redis.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Redis & Trino - Real-time indexed SQL queries (new connector)"
+author: "Allen Terleto, Julien Ruaux, Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Redis.png
+show_pagenav: false
+---
+
+Ever since the pandemic, the world has changed. The need for a digital first
+economy rose dramatically during the lockdowns. It has become clear that
+digital's impact on future businesses will be more and more important. As Redis'
```

Comment:
I think we can probably scrap these first two sentences and bring the scope in a little bit. You're kind of burying the lede by starting with such broad statements.